content/pipeline/scripting/jsonnet.md

```diff
@@ -108,3 +108,5 @@ yaml: line 1: mapping values are not allowed in this context
 ```
 
 The second most common root cause for this issue is when you forget to rename your file with a jsonnet extension. Drone assumes configuration files are written in yaml unless you explicitly use the jsonnet file extension.
+
```

Comment:
Thanks for taking the time to send the PR.  I think we should amend the above paragraph to be more clear since it is attempting to say the same thing.  I'm not sure if this does the trick, but here is an example:

```diff
-The second most common root cause for this issue is when you forget to rename your file with a jsonnet extension.
+The second most common root cause for this issue is when you forget to rename your file with a jsonnet extension in your repository and in your repository settings screen in Drone
```

Please note that whether or not a repository was active before migrating to jsonnet is irrelevant;  the repository settings will always default to .drone.yml and therefore will always need to be changed to .drone.jsonnet.  I can see how the current docs could cause some confusion, but I also want to be sure we accurately document the behavior.

Reply:
Yeah, I agree. When I re-read the paragraph above now, I see that it says the same thing, just doesn't mention specific steps to follow.

Regarding repo being active or not — that's also true. I've discovered it later after this PR.

I fixed the PR per your suggestion. I was actually hoping that somebody from the team could help me phrase it better. Thank you!